app/src/main/java/com/github/geohunt/app/data/network/firebase/models/FirebaseClaim.kt

```diff
@@ -0,0 +1,14 @@
+package com.github.geohunt.app.data.network.firebase.models
+
+import com.github.geohunt.app.model.database.api.Location
+
+internal data class FirebaseClaim(

```

Comment:
There is no use for the `internal` restriction here: it's going to create more pain than anything else (especially in tests).
```suggestion
data class FirebaseClaim(
```

Reply:
Nice catch

Reply:
Not if test are in the same package. Personally I would rather keep it internal because I think we should use it outside the "internals" of the firebase. I have seen many bug of people using method that "should have been" internalized outside of their scope. (Aka reinstating the database). However this does not applies here so I will apply your selection.